packages/integration_test/README.md

```diff
@@ -47,28 +48,27 @@ import 'dart:async';
 import 'package:integration_test/integration_test_driver.dart';
 
 Future<void> main() async => integrationDriver();
-
 ```
 
 To run a example app test with Flutter driver:
 
-```
+```sh
 cd example
```

Comment:
I'm not really sure what this means... Is it referring to the example app in `packages/integration_test/example`? If so we should update the file names accordingly, and perhaps we should move the following docs there and direct users there from here.

Reply:
Yes, it's that directory.

Reply:
Filed #64690, will follow up separately